hubblestack/extmods/fdg/command_line_parser.py

```diff
@@ -129,6 +129,9 @@ def parse_cmdline(params=None, chained=None, chained_status=None):
                     ret_match_list.extend(match_list)
                     break
 
+        if not ret_match_list:
+            log.info("No Match was found for %s in %s, returning False", key_aliases, command_line)
+            return False, ret_match_list
```

Comment:
Its wrong to return False here. It represents module did not execute, like in error conditions.
This is what fdg module does.

But here, return value is based on expectation.

Reply:
Agreed, removed this code.